### ISSUE TITLE
Parent and previous sibling content not respected on scrollback

### ISSUE THREAD

**Issue by cycle4passion** [bagusindrayana/scrollpage-js#4]
Thank you for your effort. I appreciate the small, and not overly complicated package. I think it is a few minor improvements from being a valuable resource. Nice job.

but... I found another issue.

[Codepen](https://codepen.io/c4p/pen/LYOgVGa)

If in a layout such as
```js
<div>PARENT
  <div>PREVIOUS SIBLING</div>
  <div id="main-page">
    <div  id="page1"></div>
    <div  id="page2"></div>
    <div  id="page3"></div>
  </div>
</div>
```

scrolling down, then back up does not show PARENT or PREVIOUS SIBLING and they stay inaccessible. I probably is reasonable to require main-page to be first child below a parent, but the parent should be respected (pun intended).

**Comment by bagusindrayana** [comment]
thank you i already update [demo page ](https://scrollpage-js.netlify.app/) with header and footer is outside from "main-page", but I haven't released the latest version yet i need to fix and add some other features, i also add some another example like [scroll content](https://scrollpage-js.netlify.app/examples/scroll-content.html) and [dynamic height](https://scrollpage-js.netlify.app/examples/dynamic.html)

**Comment by bagusindrayana** [comment]
fix in [v0.1.4](https://github.com/bagusindrayana/scrollpage-js/releases/tag/v0.1.4)
